docs/creating_rbac_groups_and_groupsync.md

```diff
@@ -0,0 +1,142 @@
+
+To achieve this effectiely in our experience, we needed to have development squads share access to a common "Clusterpool Host" Hub cluster with sufficient permissions to create/consume/delete clusterpools without interfering with one anothers' resources or leaking credentials cross-squad.  We accomplish this encapsulation by granting squads namespace-scoped access with each squad owning and operating within a single namespace.  This can be accomplished by creating and allocated a few roles, overviewed below.   
+
+We mirror/generate these RBAC groups from our `open-cluster-management` GitHub organization using the [group-sync operator](https://github.com/redhat-cop/group-sync-operator) after configuring a GitHub OAuth Provider on the cluster, which we'll also overview below.  
+
+## "Multitenancy" via Namespace-Scoped RBAC Roles
+
+When it comes to our internal development squads, we will sometimes take the "easy route" and allocate each sqaud (represented as an RBAC group in OpenShift mirrored from GitHub via group-sync) a namespace and give the group `cluster-admin` within that namespace, but we don't recommend that most users give anyone `cluster-admin`, even on a namespace.  Instead, we recommend allocating `create` and `delete` access sparingly.  
+
+We recommend that you create a namespace for each rbac group and grant the group the standard openshift view role and a custom clusterpool-focused role on that namespace using the ClusterRole and ClusterRoleBindings below:
+
+Custom ClusterRole - grants access to create/delete clusterpools, claims, and deployments as well as secrets (required for cloud platform creds and install-configs):
+```
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: clusterpool-user
+rules:
+  - apiGroups:
+      - 'hive.openshift.io'
+    resources:
+      - clusterclaims
+      - clusterdeployments
+      - clusterpools
+    verbs:
+      - get
+      - watch
+      - list
+      - create
+      - delete
+  - apiGroups:
+      - ''
+    resources:
+      - secrets
+    verbs:
+      - get
+      - watch
+      - list
+      - create
+      - delete
+```
+
+ClusterRoleBindings - `clusterpool-user` for access to create/delete clusterpools and claims, `view` to view clusterimagesets and secrets, and `hive-cluster-pool-user` to [propogate permissions from pools in the namespace to the ClusterProvision/Deployment namespaces for provision failure debugging](https://github.com/openshift/hive/blob/master/docs/clusterpools.md#managing-admins-for-cluster-pools):
```

Comment:
propogate --> propagate

The length of this link looks too long. Perhaps:
...propagate permissions from pools in the namespace to the ClusterProvision/Deployment namespaces for provision failure debugging (see Hive documentation on [Managing admins for Cluster Pools](https://github.com/openshift/hive/blob/master/docs/clusterpools.md#managing-admins-for-cluster-pools))